src/Shared/Recruit.Vacancies.Client/Infrastructure/Mongo/MongoDbCollectionBase.cs

```diff
@@ -47,6 +47,7 @@ private Polly.Retry.RetryPolicy GetRetryPolicy()
         {
             return Policy
                     .Handle<MongoCommandException>()
+                    .Or<MongoConnectionException>()
```

Comment:
Do we just make it MongoException to cover anything that inherits from it?

Reply:
didn't notice that 🤦‍♂️  but yes, will change